codechecker_lib/session_manager.py

```diff
@@ -305,29 +310,22 @@ def __init__(self):
         LOG.debug('Loading session config')
 
         # Check whether user's configuration exists.
-        session_cfg_file = os.path.join(os.path.expanduser("~"),
+        user_home = os.path.expanduser("~")
+        session_cfg_file = os.path.join(user_home,
                                         ".codechecker_passwords.json")
-        if not os.path.exists(session_cfg_file):
-            LOG.info("CodeChecker authentication client's example "
-                     "configuration file created at " + session_cfg_file)
-            shutil.copyfile(os.path.join(os.environ['CC_PACKAGE_ROOT'],
-                                         "config", "session_client.json"),
-                            session_cfg_file)
-            os.chmod(session_cfg_file, stat.S_IRUSR | stat.S_IWUSR)
-
         LOG.debug(session_cfg_file)
 
         scfg_dict = load_session_cfg(session_cfg_file)
 
-        if not scfg_dict["credentials"]:
+        if not scfg_dict.get("credentials"):
             scfg_dict["credentials"] = {}
 
         self.__save = scfg_dict
         self.__autologin = scfg_dict.get("client_autologin") \
             if "client_autologin" in scfg_dict else True
 
         # Check and load token storage for user
-        self.token_file = os.path.join(tempfile.gettempdir(), ".codechecker_" +
+        self.token_file = os.path.join(user_home, ".codechecker_" +
                                        getpass.getuser() + ".session.json")
         LOG.debug(self.token_file)
         if os.path.exists(self.token_file):
```

Comment:
Below this line, are we sure about what the output is talking about? This part of the code is supposed to load the token storage (which is, just like session cookies, expected to be volatile), but the permission setting code warns about the `session_cfg_file` instead.